xbgpack1.txt

```diff
@@ -0,0 +1,12 @@
+[Section]
+ Name = Super Mario 64 Backgrounds!
+ Version = 1
+ License = CC0
+ Description = Super Mario 64 based backgrounds for ReactOS. 1024x1024 each.
+ Size = 7.36 MB
+ Category = 15

```

Comment:
```suggestion
 Category = 3
```
I think this category is `ENUM_CAT_GRAPHICS`.

Reply:
I don't know if that makes sense. The graphics category is used for programs that have to do with graphics and image editing. putting backgrounds/wallpapers there would seem a bit out of place, don't you think?

Reply:
OK. I reject my suggestion.